examples/worlds/mecanum_drive.sdf

```diff
@@ -0,0 +1,365 @@
+<?xml version="1.0" ?>
```

Comment:
I renamed the other file in bcfb881 to `skid_steer_mecanum`. I think it's still interesting, since it uses the diff drive plugin for all the wheels just like the `diff_drive_skid` example world and behaves like a skid steer, but I think it has a smoother response. I think it's interesting, but I can delete it if it's too much noise